### DISCUSSION TITLE
Struggling to get started

### DISCUSSION THREAD

**Discussion by dougthinkiq** [mpostol/ASMD#266] [Q&A] [answered]
I am very interested in this work.  The first step to to get started is "Download and run the file UA.ModelDesigner.exe".  I am feeling stupid, where can I find this file?

**Comment by dougthinkiq** [comment]
Ignore the above, it's a release asset.

**Comment by mpostol** [comment, marked as answer]
@dougthinkiq thanks for the feedback. I will improve the release description.
